tests/TryCatchTest.php

```diff
@@ -142,7 +141,7 @@ function test(): string {
 
                     try {
                         $var = test();
-                    } catch (Exception $e) {
+                    } catch (Throwable $e) {
```

Comment:
If I get this right, a catch(Exception) is no longer seen as a "catch all" part?

It's technically true, but do we want to add potentially lots of false positive for a few true positives?

Reply:
Actually I need to go through and undo most of those, I'd planned to and forgot about it. `Throwable` is the "catch all" instead of `Exception`, but I changed those back on a previous attempt where I handled it incorrectly. The "catch all" thing only actually matters for `finally`.